lib/routes/initial_walkthrough/mnemonics/widgets/restore_form.dart

```diff
@@ -119,8 +124,12 @@ class RestoreFormPageState extends State<RestoreForm> {
     return null;
   }
 
-  FutureOr<List<String>> _getSuggestions(pattern) {
+  FutureOr<List<String>> _getSuggestions(String pattern, int itemIndex) {
     var suggestionList = WORDLIST.where((item) => item.startsWith(pattern)).toList();
+    if (pattern.length == 4 && suggestionList.isNotEmpty) {
+      _selectSuggestion(suggestionList.first, itemIndex);
+      return List.empty();
+    }
     return suggestionList.isNotEmpty ? suggestionList : List.empty();
```

Comment:
```suggestion
   return suggestionList
```

Reply:
Tested with this. Did not autocomplete or fill out the word.